src/main/java/cz/muni/fi/pa165/bluebat/entity/Trip.java

```diff
@@ -27,32 +16,37 @@
 
 @Entity
 @Setter
-@ToString
+@Getter
 public class Trip {
 
     @Id
     @GeneratedValue(strategy= GenerationType.IDENTITY)
-    @Getter
     private Long id;
 
-    @Getter
+    // Blank checks if not null or blank
+    // Column generates database column with notnull
+    @Column(nullable = false)
+    @NotBlank
     private String name;
 
     @Column(nullable = false)
-    @Getter
+    @NotNull
```

Comment:
Asi trochu zbytečné mít @Column(nullable = false) a @NotNull, když obojí dělá vlastně to stejné 🙂

Reply:
Tiez som si myslel ale @Column(nullable = false) nastavi riadok ze nesmie byt nulll a @NotNull kontroluje ci je null.

Reply:
Každá z anotací používá jiný přístup, ale nemyslím si, že je třeba mít tam obě. https://www.baeldung.com/hibernate-notnull-vs-nullable